testing-services/src/main/java/io/stargate/testing/TestingServicesActivator.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.testing;
+
+import io.stargate.auth.AuthorizationProcessor;
+import io.stargate.core.activator.BaseActivator;
+import io.stargate.testing.auth.LoggingAuthorizationProcessorImpl;
+import java.util.Collections;
+import java.util.Hashtable;
+import java.util.List;
+
+public class TestingServicesActivator extends BaseActivator {
```

Comment:
Maybe I missed it but what prevents this from being pulled into the actual deployment?

Reply:
The `testing-services` bundle is not put into `stargate-lib` during the main build sequence. It's copied there only during integration tests: https://github.com/stargate/stargate/pull/554/files#diff-6eb247058d9d4d2db49e65e071f2595396f06165f3ee3eb78477c403242b3353R366